.github/workflows/assets.yaml

```diff
@@ -0,0 +1,70 @@
+name: Assets
+
+on:
+  pull_request:
+
+permissions:
+  contents: write
+
+env:
+  GH_ACCESS_TOKEN: ${{ secrets.USER_PAT }}:${{ secrets.PAT }}
+
+jobs:
+  bump_and_tag:
+    runs-on: [self-hosted, small]
+    if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' && (contains(github.event.pull_request.title, 'Bump assets') || contains(github.event.pull_request.title, 'Bump banned-assets') )}}
```

Comment:
When dependabot (github.actor) opens a PR (github.event_name) that contains in its title "Bump assets" or "Bump banned-assets" it will be merged automatically

Reply:
After that, the pipeline will checkout the main branch, build, commits & pushes and finally creates a tag